### ISSUE TITLE
Request for Training Loss Curves (CE Loss & MSE Loss)

### ISSUE THREAD

**Issue by Zhuocheng0579** [multimodal-reasoning-lab/Bagel-Zebra-CoT#9]
Hi there 👋

Thanks for the great work on this project!  
I’d like to kindly ask if the authors could provide the training loss curves for both **cross-entropy loss (CE)** and **mean squared error loss (MSE)**. Having these curves would be very helpful for:

- Verifying training convergence
- Reproducing results
- Comparing different setups or hyperparameters

If the logs are available, please consider uploading them or plotting the curves. Thanks a lot!

**Comment by LeonLixyz** [comment]
Hi thanks for your interests!

While we don't have the complete log history due to switching computing devices, here are the CE loss and MSE loss curves for the first 800 steps:

![Image](https://github.com/user-attachments/assets/62b2f2a6-d623-4c61-a1cf-2c86c5105247)

![Image](https://github.com/user-attachments/assets/a608c500-d253-416b-8dad-318d4979a3b7)

**Comment by Zhuocheng0579** [comment]
Thanks a lot!
